ui/fisher/seabase.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+class SeaBase {
+    constructor() {
+        this._dbName = 'seabase';
+        this._dbVersion = 1;
+        this._storeName = 'catches';
+        this.db = null;
+    }
+
+    getConnection() {
+        let _this = this;
+
+        return new Promise(function(resolve, reject) {
+            let req = window.indexedDB.open(_this._dbName, _this._dbVersion);
```

Comment:
It does persist between sessions. I'm not sure exactly where. I believe this is somewhat of a first-class usability thing for the CEF, because hibiyasleep maintains OverlayPlugin and their own overlay, kagerou, uses localstorage to persist settings: https://github.com/hibiyasleep/kagerou/blob/master/share/lib/config.js#L512

I can't seem to track down where the actual disk storage is, though. It could be in the ACT folder itself, which I set ownership permissions on to allow updating and editing addons etc without hitting a UAC prompt. I feel like if kagerou required you to reload settings every time you started ACT I'd see more complaining about it though. So I'm not sure how it's working, just that it does (for me) and it probably works for people who are using kagerou.

Reply:
Oh, and re: the crowdsourcing, I have thought about it and would like to do that eventually, it's just a much bigger scope to add in a website side of it and a submission mechanism, and devising some kind of error-checking mechanism on the server side for it. That's definitely a longer-term goal for it though, which is why I'm hanging onto extra information (snagging) that I'm not currently using.